examples/custom-gql-apis/src/python.rs

```diff
@@ -0,0 +1,16 @@
+use pyo3::exceptions::PyRuntimeError;
+use pyo3::prelude::*;
+use raphtory_core::py_algorithm_result_base;
+use raphtory_core::python::types::repr::{Repr, StructReprBuilder};
+use raphtory_graphql::python::server::server::PyGraphServer;
+use raphtory_graphql::python::server::take_server_ownership;
+use crate::hello_world::HelloWorld;
+
+#[pyfunction(name = "add_custom_gql_apis")]
+pub fn py_add_custom_gql_apis(
+    server: PyRefMut<PyGraphServer>,
+) -> Result<PyGraphServer, PyErr> {
+    let server = take_server_ownership(server)?;
```

Comment:
Can this fail?

Reply:
yes if you do

```
server = ???
py_add_custom_gql_apis(server)
py_add_custom_gql_apis(server)
```
In the second call, the server would be already consumed. To be fair, we should change the server API to be mutable, as we did for the vector API